test_results_win64.csv

```diff
@@ -17,69 +17,69 @@ PVWatts,Host Developer,943704.187500,14.690550,15.743791,;
 PVWatts,Single Owner,87966056,7.918575,8.357705,; 
 PVWatts,Merchant Plant,87966056,7.181620,NA,; 
 PVWatts,Leveraged Partnership Flip,87966056,8.091964,8.778845,; 
-PVWatts,All Equity Partnership Flip,87966056,9.820335,11.885724,; 
+PVWatts,All Equity Partnership Flip,87966056,0,0,; 
```

Comment:
LCOE and LPPA should not be zero. This suggests a bug somewhere.

Reply:
Chunk of code calculating cf_energy_sales and cf_energy_purchases for each year was not brought into ssc/cmod_equpartflip.cpp. The LCOE and LPPA are no longer zero for this financial model.